shared/views/countries.view.lkml

```diff
@@ -0,0 +1,26 @@
+view: countries {
+  sql_table_name: `mozdata.static.country_codes_v1` ;;
+
+  dimension: name {
+    type: string
+    sql: ${TABLE}.name ;;
+    description: "Official country name per ISO 3166"
+  }
+
+  dimension: code {
+    type: string
+    sql: ${TABLE}.code ;;
+    description: "ISO 3166 alpha-2 country code"
+  }
+
+  dimension: tier {
+    type: string
+    sql:
+      CASE
+      WHEN
+        ${TABLE}.code IN ('US', 'CA', 'DE', 'FR', 'GB') THEN 'tier-1'
+      ELSE 'non-tier-1'
```

Comment:
I'm also open to renaming this to `Tier 1` or something else.

Reply:
I think the goal for this view should be labels that look nice on charts, so I think "Tier 1" is more appropriate than "tier-1".

I've seen lots of redash queries that are labeled for "ROW" (rest of world) as the counterpart to "Tier 1". I do find the ROW term a bit dismissive, which is problematic. "Other" might be good here.

@xluo-ds - You've done some work around defining more granular tiers for different business contexts. Do you see codifying a reusable tier definition here problematic given that work. Any suggestions for terminology here?

Reply:
@jklukas  The tier-structure work is still in process, and hopefully in a better shape toward the end of September. In this case, I think we can use either (1) 'Tier 1' and 'non Tier 1' or (2) 'Tier 1' and 'Other'. However, the specific meaning/understanding of "Tier 1" may vary among different products so it would be great if we make annotation available in Looker dashboard.